packages/slate-editor/src/components/EditorBlock/ResizableEditorBlock.tsx

```diff
@@ -98,53 +95,59 @@ export const ResizableEditorBlock = forwardRef<HTMLDivElement, Props>((props, re
                 {renderInjectionPoint(renderReadOnlyFrame ?? renderEditableFrame, props)}
-                {resizable && props.isSelected && (
-                    <Draggable.DraggableCore
-                        offsetParent={blockElement ?? undefined}
-                        onDrag={handleResizeEvent}
-                        onStart={handleResizingStarted}
-                        onStop={handleResizingFinished}
-                    >
-                        <div contentEditable={false}>
-                            <ResizeButton
-                                className={classNames(
-                                    styles.ResizeButton,
-                                    isInvertedResizing ? styles.left : styles.right,
-                                )}
-                                position={isInvertedResizing ? 'left' : 'right'}
-                            />
-                        </div>
-                    </Draggable.DraggableCore>
+                {resizable && (props.isSelected || props.isHovered) && (
+                    <div contentEditable={false}>
+                        <ResizeButton
+                            offsetParent={blockElement ?? undefined}
+                            onDrag={handleResizeEvent}
+                            onStart={handleResizingStarted}
+                            onStop={handleResizingFinished}
+                            className={classNames(styles.ResizeButton, styles.left)}
+                            position={'left'}
+                        />
+                        <ResizeButton
+                            offsetParent={blockElement ?? undefined}
+                            onDrag={handleResizeEvent}
+                            onStart={handleResizingStarted}
+                            onStop={handleResizingFinished}
+                            className={classNames(styles.ResizeButton, styles.right)}
+                            position={'right'}
+                        />
+                    </div>
                 )}
             </>
         );
     }
 
     return (
-        <EditorBlock
-            {...attributes}
-            align={align}
-            ref={mergeRefs(setBlockElement, ref)}
-            renderBelowFrame={({ isSelected }) => (
-                <>
-                    {sizer}
-                    {renderInjectionPoint(renderBelowFrame, { isSelected })}
-                </>
-            )}
-            renderEditableFrame={renderEditableFrame ? renderFrame : undefined}
-            renderReadOnlyFrame={renderReadOnlyFrame ? renderFrame : undefined}
-            renderMenu={isResizing ? undefined : renderMenu}
-            selected={isResizing || undefined}
-            width={
-                resizable
-                    ? toString(convert(Size(pixelWidth, Unit.PIXELS), unit(width), containerWidth))
-                    : width
-            }
-        />
+        <div style={{ pointerEvents: isResizing ? 'none' : undefined }}>
```

Comment:
I see weird behavior change:

1. When I upload new images, they initially appear `full-width` (but should appear `constrained`).
2. There is no vertical spacing between images.

I suspect it's because of this structure decision.

![image](https://user-images.githubusercontent.com/370680/197744342-9715ed36-4d0b-4fb5-a4f1-ba4a8297d949.png)